.github/workflows/build.yml

```diff
@@ -442,3 +442,101 @@ jobs:
       with:
         name: WasmEdge-${{ needs.get_version_v2.outputs.version }}-android_aarch64.tar.gz
         path: build/WasmEdge-${{ needs.get_version_v2.outputs.version }}-Android.tar.gz
+        
+  build_openwrt:
```

Comment:
I found this workflow will take 1.5 hours to build. So it's better to split into a standalone workflow, not in the original build workflow.